src/main/webapp/intrigue-api/metacards/metacards.js

```diff
@@ -294,28 +294,30 @@ const metacardsByTag = async (parent, args, context) => {
 }
 
 const metacardsById = async (parent, args, context) => {
-  return args.ids.map(id =>
-    metacards(
-      parent,
-      {
-        filterTree: {
-          type: 'AND',
-          filters: [
-            {
-              type: '=',
-              property: 'id',
-              value: id,
-            },
-            {
-              type: 'LIKE',
-              property: 'metacard-tags',
-              value: '%',
-            },
-          ],
+  return await Promise.all(
+    args.ids.map(id =>
+      metacards(
+        parent,
+        {
+          filterTree: {
+            type: 'AND',
+            filters: [
+              {
+                type: '=',
+                property: 'id',
+                value: id,
+              },
+              {
+                type: 'LIKE',
+                property: 'metacard-tags',
+                value: '%',
+              },
+            ],
+          },
+          settings: args.settings,
```

Comment:
Instead of creating a map of promises here, can we use a single promise and concat the ids in an OR filter so we only make one network request? Something like: 
```
{
    type: 'OR'
    filters: args.ids.map(id=>({type: '=', property: 'id', id})
}
```